avr/platform.txt

```diff
@@ -188,3 +188,20 @@ tools.ctags.path={runtime.tools.ctags.path}
 tools.ctags.cmd.path={path}/ctags
 tools.ctags.pattern="{cmd.path}" -u --language-force=c++ -f - --c++-kinds=svpf --fields=KSTtzns --line-directives "{source_file}"
 
+# The following recipes enable adding code to specific places
+# of the preprocessed firmware sketch. This is done by pasting 
+# the content of two files
+#   kaleidoscope_internal/sketch_preprocessing/sketch_header.h_,
+#   kaleidoscope_internal/sketch_preprocessing/sketch_footer.h_
+# to the top and the bottom of the preprocessed sketch file.
+#
+# These two files will make the include files
+#   kaleidoscope_internal/sketch_preprocessing/sketch_header.h and
+#   kaleidoscope_internal/sketch_preprocessing/sketch_footer.h
+# being included during the firmware compile.
```

Comment:
Why have the `.h` and `.h_` files both? What problems does concatenating the `.h` files directly cause?

Reply:
Nothing. I just found it to be more verbose when looking at the preprocessed sketch. There, the includes lead the way to where the code came from.

But I am also ok with including directly.